### DISCUSSION TITLE
I found this document is missing. 404

### DISCUSSION THREAD

**Discussion by LintaoAmons** [michaelb/sniprun#257] [General]
<img width="853" alt="image" src="https://github.com/michaelb/sniprun/assets/95092244/46d57add-8211-4400-8955-c622bfea4e70">

**Comment by michaelb** [comment]
Fixed link, though there is no much there yet (and I realize, not many explanations either)

**Reply by LintaoAmons** [reply]
Thx~ I'm trying to run hurl with sniprun~ So trying to figuring out how to configure a interpreter

**Reply by michaelb** [reply]
Don't hesitate to ask for precision/doc/example, I'm aware it's currently lacking

Don't hesitate to report your findings so they get added to the community examples
